### PR TITLE
libu2f-host: new port at 1.1.3

### PR DESCRIPTION
**Pull request by l2dy**

### CODE REVIEW

File: security/libu2f-host/Portfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+long_description    Libu2f-host provides a C library and command-line tool \
+                    that implements the host-side of the U2F protocol. There \
+                    are APIs to talk to a U2F device and perform the U2F Register \
+                    and U2F Authenticate operations.
+
+homepage            https://developers.yubico.com/${name}/
+master_sites        https://developers.yubico.com/${name}/Releases/
+
+extract.suffix      .tar.xz
```
Comment (mojca) [inline security/libu2f-host/Portfile]: This is usually done with
```tcl
use_xz                  yes
```
but I don't have any explanation or any reason against your approach. I'm not sure if you should listen to me in this case.

Comment (g5pw) [inline security/libu2f-host/Portfile]: Yeah, `use_xz yes` is the canonical way of specifying archive type.

File: security/libu2f-host/Portfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libu2f-host
+version             1.1.3
+categories          security
+platforms           darwin
+maintainers         nomaintainer
```
Comment (g5pw) [inline security/libu2f-host/Portfile]: I think the general policy is against accepting nomaintainer submissions unless it's a dependency of a maintained port

**Comment by l2dy** [comment]
@g5pw Both problems were already fixed.

**Comment by g5pw** [comment]
Oh, never mind, sorry

**Pull request merged**